### ISSUE TITLE
The "Table Options Menu" (F12) shows rotated 90 degrees in VR

### ISSUE THREAD

**Issue by KrakenByteKraken** [vpinball/vpinball#1737]
When you press F12 to show the "Table Options Menu" it appears with rotated 90 degrees. It happens with "**VPinballX_GL-10.8.0-1954-aba1618-Release-win-x64**". As shown in picture:

![20240527_094709_MixedReality](https://github.com/vpinball/vpinball/assets/117071087/8b32a2ee-d9eb-41ce-8bce-0befe801dfa2)

I noticed that it was because the "View mode" is "Cabinet" in Preferences -> "Video/Graphic Options":

![imagen](https://github.com/vpinball/vpinball/assets/117071087/7fff93d2-b4d0-4806-98f4-5aa38cfb8cfc)

But if you change this "View mode" to "Desktop (No FSS)", and now it works as I expected:

![imagen](https://github.com/vpinball/vpinball/assets/117071087/2c83ca01-4400-47f1-8017-92f688ccf9d7)

![20240527_095948_MixedReality](https://github.com/vpinball/vpinball/assets/117071087/17073364-d3fe-4f6b-953e-fae296d5311a)

I don't know if this behavior is the desired by the Developer Team, but I think that, when you are in VR, this setting must be independent of the setting in view mode of Video Graphic Options Menu.

Another thing related with this one, is that tables wicht read the parameter form this menu render or not some items in VR like side rails or lockdown bar: They dissapears in VR because it thinks that you are in cabinet (example "Batman [The Dark Knight] (Stern 2008) bord mod  Version 1.16" ([https://vpuniverse.com/files/file/11451-batman-the-dark-knight-stern-2008-bord-mod/](https://vpuniverse.com/files/file/11451-batman-the-dark-knight-stern-2008-bord-mod/)). 

In other words: To make this works fine in VR, you must set up your config to play in desktop mode but I think this could be made by VPX automatically.

Anyway, thank you to all involved in this amazing hobby.
